build/Targets/VSL.Imports.targets

```diff
@@ -74,6 +74,38 @@
     </PropertyGroup>
   </Target>
 
+  <!-- ====================================================================================
+
+         Generation of binding redirects.
+
+       ==================================================================================== -->
+  <ItemGroup Condition="'$(GenerateMSBuildBindingRedirects)' == 'true'">
+      <SuggestedBindingRedirects Include="Microsoft.Build, Version=$(VisualStudioReferenceAssemblyVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+        <MaxVersion>$(VisualStudioReferenceAssemblyVersion)</MaxVersion>
+      </SuggestedBindingRedirects>
+      <SuggestedBindingRedirects Include="Microsoft.Build.Framework, Version=$(VisualStudioReferenceAssemblyVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+        <MaxVersion>$(VisualStudioReferenceAssemblyVersion)</MaxVersion>
+      </SuggestedBindingRedirects>
+      <SuggestedBindingRedirects Include="Microsoft.Build.Tasks.$(MSBuildAssemblyNameFragment), Version=$(VisualStudioReferenceAssemblyVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+        <MaxVersion>$(VisualStudioReferenceAssemblyVersion)</MaxVersion>
+      </SuggestedBindingRedirects>
+  </ItemGroup>
+  <PropertyGroup>
+    <ForceGenerationOfBindingRedirects Condition="'$(GenerateMSBuildBindingRedirects)' == 'true'">true</ForceGenerationOfBindingRedirects>
+    <AutoGenerateBindingRedirects Condition="'$(ForceGenerationOfBindingRedirects)' == 'true'">true</AutoGenerateBindingRedirects>
```

Comment:
Having `GenerateMSBuildBindingRedirects`, `AutoGenerateBindingRedirects` and `ForceGenerationOfBindingRedirects` all at once is making my head spin. Can we reduce by one or two? Any reason I always wouldn't want to run the force target below?